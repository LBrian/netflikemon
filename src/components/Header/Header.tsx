function Header() {
  return (
    <div className='btm-nav-lg short:btm-nav-sm navbar bg-red-500 text-white text-2xl short:text-xl font-bold sticky top-0 z-10'>
      <div className='px-4'>{document.title}</div>
    </div>
  )
}

export default Header
