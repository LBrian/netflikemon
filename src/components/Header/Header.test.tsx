import { render } from '@testing-library/react'
import Header from './Header'

test('renders Header component', () => {
  expect(render(<Header />)).toMatchSnapshot()
})
