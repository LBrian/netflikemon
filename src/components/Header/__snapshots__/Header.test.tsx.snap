// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="btm-nav-lg short:btm-nav-sm navbar bg-secondary text-secondary-content text-2xl short:text-xl font-bold sticky top-0 z-10"
      >
        <div
          class="px-4"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="btm-nav-lg short:btm-nav-sm navbar bg-secondary text-secondary-content text-2xl short:text-xl font-bold sticky top-0 z-10"
    >
      <div
        class="px-4"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
